relayer-cli/src/application.rs

```diff
@@ -148,7 +148,17 @@ impl Application for CliApp {
 
         let config = config_path
             .map(|path| self.load_config(&path))
-            .transpose()?
+            .transpose()
+            .map_err(|err| {
+                eprintln!("The Hermes configuration file is invalid, reason: {}", err);
```

Comment:
Just wanted to double-check: using `tracing::error` here is not an option here, right (because it's not initialized yet)? If it is an option, we should consider using that.

Reply:
Exactly, I tried to use it before but then realized that it needs to get the log level from the config file for initialization. The actual log init is done later in the same function, so I don't think it would be possible to use it here.